scripts/remove-root-items

```diff
@@ -3,7 +3,7 @@
 # remove-root-items
 # Author: Michael Stealey <michael.j.stealey@gmail.com>
 
-find /home/docker/hydroshare -user root -ls | rev | cut -d ' ' -f 1 | rev > root-items.txt
+find /hydroshare -user root -ls | rev | cut -d ' ' -f 1 | rev > root-items.txt
```

Comment:
@mjstealey I am curious about whether there would be still root items under /hydroshare after everything is run under hydro-service account. Can you confirm there are still root items or not?